pages/pipelines/environment_variables.md.erb

```diff
@@ -82,7 +82,7 @@ Example: `"s3://name-of-your-s3-bucket/$BUILDKITE_JOB_ID"`
 
 <h3 class="h3-caps">BUILDKITE_BIN_PATH</h3>
 
-The path to the `buildkite-agent` binary. The value cannot be modified as an environment variable.
+The path to the directory containing the `buildkite-agent` binary. The value cannot be modified as an environment variable.
 
 Example: `"/usr/local/bin/buildkite-agent"`
```

Comment:
I'm not  very familiar with this ENV var, but based on the new description should the example be updated to `/usr/local/bin/`?

Reply:
:man_facepalming: indeed.